2-Browsers/Week1/homework/ex1-bookList/style.css

```
@@ -1 +1,20 @@
 /* Write your style here */
+ul {
+  display: flex;
```

Maybe we can add width here to make it looks nicer:
```suggestion
  display: flex;
  width: calc( 100% - 41px);
```

---

i implemented this changes 👍